src/app/ApplicationTemplate.Shared/Presentation/Diagnostics/DiagnosticsOverlayViewModel.cs

```diff
@@ -19,7 +19,9 @@ public partial class DiagnosticsOverlayViewModel : ViewModel
 
 		public IDynamicCommand NavigateToDiagnosticsPage => this.GetCommandFromTask(async ct =>
 		{
+#if !PRODUCTION
 			await this.GetService<ISectionsNavigator>().OpenModal(ct, () => new DiagnosticsPageViewModel());
+#endif
```

Comment:
Wouldn't it be better to hide the whole diagnostic overlay instead?

Reply:
This command is triggered when you tap on the Splashscreen so even if you hide the whole diagnostics overlay on PROD (which is already the case), it will be triggered when the user will tap 3 times while the splashscreen is displayed.